### PR TITLE
add deeplinks vars in nginz config of wire-server prod-values example

### PR DESCRIPTION
**Pull request by amitsagtani97**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

_Briefly describe the issue you have solved or implemented with this pull request. If the PR contains multiple issues, use a bullet list._

### Causes (Optional)

_Briefly describe the causes behind the issues. This could be helpful to understand the adopted solutions behind some nasty bugs or complex issues._

### Solutions

_Briefly describe the solutions you have implemented for the issues explained above._

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: values/wire-server/prod-values.example.yaml [hunk 1]
```diff
@@ -192,6 +192,19 @@ nginz:
...
+    deeplink:
+      endpoints:
+        backendURL: "https://nginz-https.example.com"
+        backendWSURL: "https://nginz-ssl.example.com"
+        teamsURL: "https://teams.example.com"
+        accountsURL: "https://account.example.com"
+        blackListURL: "https://clientblacklist.wire.com/prod"
+        websiteURL: "https://wire.com"
+      apiProxy: # (optional)
+        host: "socks5.proxy.com"
```
Comment (julialongtin) [inline values/wire-server/prod-values.example.yaml:204]:
```suggestion
        ##### Uncomment these if you want to direct your users into using a proxy 
        # host: "socks5.proxy.com"
```

File: values/wire-server/prod-values.example.yaml [hunk 2]
```diff
@@ -192,6 +192,19 @@ nginz:
...
+        port: 1080
```
Comment (julialongtin) [inline values/wire-server/prod-values.example.yaml:205]:
```suggestion
        # port: 1080
```

File: values/wire-server/prod-values.example.yaml [hunk 3]
```diff
@@ -192,6 +192,19 @@ nginz:
...
+        needsAuthentication: true
```
Comment (julialongtin) [inline values/wire-server/prod-values.example.yaml:206]:
```suggestion
        # needsAuthentication: true
```

**Review by julialongtin — CHANGES_REQUESTED** [review]

File: values/wire-server/prod-values.example.yaml [hunk 4]
```diff
@@ -192,6 +192,20 @@ nginz:
...
+        teamsURL: "https://teams.example.com"
+        accountsURL: "https://account.example.com"
+        blackListURL: "https://clientblacklist.wire.com/prod"
+        websiteURL: "https://wire.com"
+      apiProxy: # (optional)
+        ##### Uncomment these if you want to direct your users into using a proxy
+        #host: "socks5.proxy.com"
+        #port: 1080
+        #needsAuthentication: true
+      title: "My Custom Wire Backend"
```
[the remark this answers is not in the document]
Comment (amitsagtani97) [reply values/wire-server/prod-values.example.yaml]: makes sense, fixed.

File: values/wire-server/prod-values.example.yaml [hunk 5]
```diff
@@ -192,6 +192,20 @@ nginz:
...
+    deeplink:
+      endpoints:
+        backendURL: "https://nginz-https.example.com"
+        backendWSURL: "https://nginz-ssl.example.com"
+        teamsURL: "https://teams.example.com"
+        accountsURL: "https://account.example.com"
+        blackListURL: "https://clientblacklist.wire.com/prod"
+        websiteURL: "https://wire.com"
+      title: "My Custom Wire Backend"
+      apiProxy: # (optional)
```
Comment (julialongtin) [inline values/wire-server/prod-values.example.yaml:204]:
```suggestion
      # Optional -- Uncomment these if you want to direct all of your mobile users to use the same proxy.
      #apiProxy:
```

File: values/wire-server/prod-values.example.yaml [hunk 6]
```diff
@@ -192,6 +192,20 @@ nginz:
...
+        ##### Uncomment these if you want to direct your users into using a proxy
```
Comment (julialongtin) [inline values/wire-server/prod-values.example.yaml:205]:
```suggestion
```

**Pull request merged**